lib/plugins/aws/invokeLocal/resolvers/snsTopic.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = {
+  'Fn::GetAtt': (plugin, resource, resourceName, parameters) => {
+    switch (parameters[1]) {
+      case 'TopicName':
+        return new Promise(resolve => {
```

Comment:
Same here

Reply:
This is needed to make sure that the resource handlers can have a ".then" action on their result. Since the version of nodejs you support does not support the async and await declorations, it is not possible to just return a value without causing inconsistency with result handling.